feature-libs/product-configurator/rulebased/components/conflict-suggestion/configurator-conflict-suggestion.component.ts

```diff
@@ -39,7 +39,7 @@ export class ConfiguratorConflictSuggestionComponent {
   displayConflictSuggestion(group: Configurator.Group): boolean {
     return group.groupType === Configurator.GroupType.CONFLICT_GROUP &&
       group.attributes
-      ? group.attributes?.length > 1
+      ? group.attributes?.length > 0
```

Comment:
Remove un-needed chaining operator
```suggestion
      ? group.attributes.length > 0
```